test/e2e/suite.go

```diff
@@ -167,32 +166,33 @@ func (t *TestSuite) IsUsingDevPorch() bool {
 		Name:      porch.Spec.Service.Name,
 	}, &service)
 
-	return service.Spec.Type == coreapi.ServiceTypeExternalName && service.Spec.ExternalName == "host.docker.internal"
+	isPorchLocal := len(service.Spec.Selector) == 0
+	areAllLocal := service.Spec.Type == coreapi.ServiceTypeExternalName
```

Comment:
What all means in this context?

Reply:
function-runner and the porch-controllers.
However now that you pointed this out. I will replace the "all" part by properly checking if function-runner is running in-cluster or not, because I think that is how the result is actually used in the end-to-end tests.
Thanks.

Reply:
done.